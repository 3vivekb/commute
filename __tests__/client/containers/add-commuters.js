/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStores} from '../../test-utils/mock-data.js'

import AddCommuters from '../../../client/containers/add-commuters'

const mockStore = makeMockStore(mockStores.init)

describe('Container > AddCommuters', () => {
  it('Add Commuters View loads (base case)', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <AddCommuters />
      </Provider>
    )
    expect(mountToJson(tree.find(AddCommuters))).toMatchSnapshot()
  })

  /* it('Add Commuters View loads (from existing commuter group)', () => {
    throw new Error('unimplemented')
  })

  it('Create Commuter Group', () => {
    throw new Error('unimplemented')
  })

  it('Append Commuters to existing commuter group', () => {
    throw new Error('unimplemented')
  }) */
})
