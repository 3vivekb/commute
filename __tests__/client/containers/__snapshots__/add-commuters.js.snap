exports[`Container > AddCommuters Add Commuters View loads (base case) 1`] = `
<Connect(AddCommuters)>
  <AddCommuters
    dispatch={[Function]}>
    <h1>
      Add Commuters
    </h1>
  </AddCommuters>
</Connect(AddCommuters)>
`;
